server/requirements.txt

```diff
@@ -9,7 +8,7 @@ flatbuffers>=1.10.0
 matplotlib>=2.2
 numpy>=1.15.2
 pandas>=0.23.1
-scanpy>=1.3.2
+scanpy>=1.3.7
```

Comment:
Why did this change? Shouldn't this be in master?

Reply:
I think you may need to rebase this branch onto master and then force push. I can help with that if you want.

Reply:
Cool. We can do that, but force push terrifies me. Can I get your help on this so I don't accidentally break the whole world? ^_^